### DISCUSSION TITLE
[Help]: E: Unable to locate package mongodb-server-core

### DISCUSSION THREAD

**Discussion by kjayGA** [mbentley/docker-omada-controller#390] [Help]
### Controller Version

v5.13.22

### Describe Your Issue or Question

Still trying to build this image...
Now I receive the following error.  "E: Unable to locate package mongodb-server-core"

### Expected Behavior

build docker image

### Steps to Reproduce

sudo docker build   --build-arg INSTALL_VER="5.13"   --build-arg ARCH="arm64"   -f Dockerfile.v5.x   -t mbentley/omada-controller:5.13-arm64 .

### How You're Launching the Container

```plain
Havent been able to build the docker image
```

### Container Logs

```plain
Cannot build image.
```

**Comment by mbentley** [comment]
Not sure why it is not building for you.  Could you provide info about your Docker engine you're building on (`docker info` output)?

My CI has been running without any issues and I just ran a build just fine locally on my macbook:

```bash
docker build --progress plain --build-arg INSTALL_VER="5.13" --build-arg ARCH="arm64" -f Dockerfile.v5.x -t mbentley/omada-controller:5.13-arm64 . &> ~/build_out.txt
```

Build output log:
[build_out.txt](https://github.com/mbentley/docker-omada-controller/files/13692991/build_out.txt)

Maybe try to make sure you're pulling the latest base image?  Try adding `--pull` to the build arguments.  Are you also trying to build the arm64 image on a non-arm64 system?  If so, you need to add the platform flag: `--platform linux/arm64`

**Reply by kjayGA** [reply]
I am thinking it has to do with trying to run Ubuntu 23.04 .  I am currently installing Ubuntu 22.04 and will attempt to build and report back.

**Reply by kjayGA** [reply]
No problems building image on Ubuntu 22.04.
